src/block/interface.ts

```diff
@@ -12,6 +12,11 @@ import type { Link, Version } from '../link/interface.js'
  */
 export interface ByteView<Data> extends Uint8Array, Phantom<Data> {}
 
+/**
+ * Similar to ByteView but extends ArrayBuffer.
+ */
+export interface ArrayBufferView<Data> extends ArrayBuffer, Phantom<Data> {}
```

Comment:
just confirming that this is exported from index.ts: 

https://github.com/multiformats/js-multiformats/blob/aa9c7303a7bd844d659163261d301c5c95f26c3b/src/index.ts#L210

and

https://github.com/multiformats/js-multiformats/blob/aa9c7303a7bd844d659163261d301c5c95f26c3b/src/interface.ts#L1-L5